fuzzing/tools/BUILD

```diff
@@ -15,10 +15,43 @@
 
 # This package contains tool scripts
 
-load("@rules_python//python:defs.bzl", "py_binary")
+load("@rules_python//python:defs.bzl", "py_binary", "py_library", "py_test")
+
+py_library(
+    name = "dict_validator_lib",
+    srcs = ["dict_validator.py"],
```

Comment:
Let's have a separate .py file with the library, and a separate tool file that provides the binary for the Bazel rule.